proto-lens-tests/tests/enum.proto

```diff
@@ -20,6 +20,13 @@ enum Baz {
   BAZ8 = 12;
 }
 
+message TwoBazs {
+  // We don't have the direct capability to generate arbitrary instances of
```

Comment:
Ah, good point.  Can you please open a PR?  (In the meantime it seems relatively straightforward to work around.)